src/test/java/org/apache/datasketches/theta/SketchTest2.java

```diff
@@ -0,0 +1,286 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.datasketches.theta;
+
+import static org.apache.datasketches.Util.DEFAULT_UPDATE_SEED;
+import static org.testng.Assert.assertEquals;
+
+import org.apache.datasketches.memory.Memory;
+import org.apache.datasketches.tuple.Util;
+import org.testng.annotations.Test;
+
+public class SketchTest2 {
```

Comment:
Could we have a more meaningful name please?

Reply:
OK, I renamed the test class as you requested. However, in looking at the tests I realized that I had all the heapify tests, but I forgot to add the "wrap" tests.  So I added them.